test/unit_tests/braket/analog_hamiltonian_simulator/test_validator/validators/test_atom_arrangement.py

```diff
@@ -8,40 +8,42 @@
 
 
 @pytest.fixture
-def atom_array_data():
+def atom_arrangement_data():
     return {
         "sites": [[0, 0], [0, 4e-6], [5e-6, 0], [5e-6, 4e-6]],
         "filling": [1, 0, 1, 0],
     }
 
 
 @pytest.fixture
-def mock_atom_array_data():
+def mock_atom_arrangement_data():
     data = {
         "sites": [],
         "filling": [1, 1, 1, 1],
     }
     return AtomArrangement.parse_obj(data).dict()
 
 
-def test_valid_atom_array(atom_array_data, device_capabilities_constants):
-    try:
-        AtomArrangementValidator(capabilities=device_capabilities_constants, **atom_array_data)
-    except ValidationError as e:
-        pytest.fail(f"Validate test is failing : {str(e)}")
+# def test_valid_atom_array(atom_arrangement_data, device_capabilities_constants):
+#     try:
+#         AtomArrangementValidator(
+#             capabilities=device_capabilities_constants, **atom_arrangement_data
+#         )
+#     except ValidationError as e:
+#         pytest.fail(f"Validate test is failing : {str(e)}")
```

Comment:
delete

Reply:
That was commented out accidentally, sorry, i will resume it.